Source/WebKit/WebProcess/Network/webrtc/RTCDataChannelRemoteManager.cpp

```diff
@@ -41,27 +42,21 @@ namespace WebKit {
 RTCDataChannelRemoteManager& RTCDataChannelRemoteManager::sharedManager()
 {
     static RTCDataChannelRemoteManager* sharedManager = new RTCDataChannelRemoteManager;
+    sharedManager->initialize();
```

Comment:
Is is right to call `initialize()` on every access?

Reply:
Ah, what a mistake. thanks.